.azure-pipelines/recover_testbed/common.py

```diff
@@ -66,20 +66,25 @@ def posix_shell_onie(dut_console, mgmt_ip, image_url):
 
                     x = x.decode('ISO-8859-9')
 
-                    if "GNU GRUB" in x:
-                        enter_onie_flag += 1
+                    if is_nexus and "loader" in x and ">" in x:
+                        dut_console.remote_conn.send('reboot\n')
                         continue
 
-                    if "SONiC-OS-" in x and enter_onie_flag == 1:
+                    # if "GNU GRUB" in x:
+                    #     enter_onie_flag += 1
+                    #     continue
+
+                    if "-OS-" in x and enter_onie_flag is True:
                         # Send arrow key "down" here.
                         dut_console.remote_conn.send(b'\x1b[B')
                         continue
 
                     if "*ONIE" in x and "Install OS" not in x:
                         dut_console.remote_conn.send("\n")
-                        enter_onie_flag += 1
+                        enter_onie_flag = False
```

Comment:
Could you create constant for the strings like "-OS-", "*ONIE", "Install OS" and others, so that it would be clear to speculate the usage from the constant name

Reply:
done